charts/kubewarden-controller/Chart.yaml

```diff
@@ -23,9 +23,9 @@ maintainers:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.0.11-rc2
+version: 2.0.11-rc3
 # This is the version of Kubewarden stack
-appVersion: v1.12.0-rc2
```

Comment:
I would leave the app version unchanged. This is just a fix of the controller, isn't it?